.github/workflows/tests.yml

```diff
@@ -71,6 +71,8 @@ jobs:
         run: yarn test
         env:
           CI: true
+      - name: Test NewRelic
```

Comment:
Why add a new command. Our CI run tests recursively for each package?

Reply:
No, `yarn test` runs the `test` script defined in the root `package.json`. And that script runs `jest` only.